sass/style.sass

```diff
@@ -1034,3 +1034,25 @@ td > .deployment-tag
 
 .button-row a
 	margin-left: 10px
+
+.create-page-git
+    border: 1px solid #dcdcdc;
+    display: block;
+    overflow: auto;
+
+    .title
+        display: block
+        line-height: 26px
+        padding: 5px
+        width: 18%
+        background: #ececec
+        border-right: 1px solid #dcdcdc
+        font-weight: bold
+        float: left
+
+    input[type="readonly"]
+        width: 82%
```

Comment:
I got this feeling that the magic number 82% is to match the page title? Can that be documented / comment?